.github/workflows/build-containers.yml

```diff
@@ -11,9 +14,14 @@ jobs:
     steps:
       - uses: actions/checkout@v2
 
+      - name: Get changed files
```

Comment:
Does this give you just the changed Dockerfiles?

Reply:
We can add a manual trigger, but I recommend we keep this workflow as being only triggered by added/modified Dockerfiles to require all builds of our container images to go through a PR process.

Reply:
i would expect this validation to be done locally first to just validate the Dockerfile. The problem however is what if we want to simply update the current `latest` packages. I think a better approach would be to use these triggers but update all our Dockerfiles to contain the explicit versions, this way we can truly have a git operation model.
This is what i meant by locking down our versions int the environments as we currently just use the latest available for most packages. If we adopt the git ops model, then every update will be clear about what the environments contain.